src/ui/screens/settings/ViewSubscriptionManagement/SubscriptionManagementItem.qml

```diff
@@ -157,71 +157,48 @@ ColumnLayout {
     }
 
     RowLayout {
-        id: relayUpsell
-        objectName: _objectName + "-relayUpsell-layout"
+        id: upgradeToAnnualSub
+        objectName: _objectName + "-upgradeToAnnualSub-layout"
 
         visible: false
         Layout.fillWidth: true
         Layout.topMargin: MZTheme.theme.listSpacing
         Layout.bottomMargin: MZTheme.theme.windowMargin
 
-        Component.onCompleted: {
-            if (visible) {
-                Glean.sample.bundleUpsellViewed.record();
-            }
-        }
-
-        ColumnLayout {
+        MZBoldInterLabel {
```

Comment:
Label should be center aligned vertically

Reply:
We seem to be getting this for free - perhaps something to do with RowLayout {}. Do you think it's worth stipulating it anyway in case there is some discrepancy in handling between platforms?

Reply:
thats because `font.pixelSize` and `lineHeight` are currently the same, so the label component is only as big as the text inside of it (no need to align), but once you [change the lineHeight ](https://github.com/mozilla-mobile/mozilla-vpn-client/pull/8199#discussion_r1348821231) to be bigger than the font, the label becomes bigger than just the text inside of it, making it necessary to vertically align